sdk/src/feature_set.rs

```diff
@@ -383,6 +383,10 @@ pub mod stake_allow_zero_undelegated_amount {
     solana_sdk::declare_id!("sTKz343FM8mqtyGvYWvbLpTThw3ixRM4Xk8QvZ985mw");
 }
 
+pub mod stake_raise_minimum_delegation_to_1_sol {
+    solana_sdk::declare_id!("4xmyBuR2VCXzy9H6qYpH9ckfgnTuMDQFPFBfTs4eBCY1");
```

Comment:
Is this a feature-proposal id?

Reply:
Yep!

Reply:
Wish there was some way to tell that from the address. Maybe include a comment that has the feature keypair address? Or can you think of a nicer way to make it clear that no one keypair can sign to activate this feature?

Reply:
Could add an assertion/test that the address is off-curve?

Reply:
Ok I've added a comment. If you'd like it tweaked I'll happily fix it in a subsequent PR.